test/common/fees/fees.spec.ts

```diff
@@ -39,6 +40,9 @@ describe('fees', function () {
         log: fakeLog
       })
 
+      // make typescript happy with this typecheck
+      if (fees.fees == null) fail()
```

Comment:
Include message parameter for `fail()`. It'll be handy later

Reply:
Fixed in https://github.com/EdgeApp/edge-currency-plugins/pull/151/commits/6a3d20c541036a50c13f1ad58532e19d8292f5cf